lib/std/zig/parse.zig

```diff
@@ -3602,7 +3595,7 @@ const Parser = struct {
         });
         _ = try p.parsePayload();
         const else_expr = try bodyParseFn(p);
-        if (else_expr == 0) return p.fail(.invalid_token);
+        assert(then_expr != 0);
```

Comment:
again: how to rule out code can trip assertion?

Reply:
Same as above.